src/views/backoffice/BackofficeVendorNew.vue

```diff
@@ -211,7 +309,27 @@ const newVendor = ref<Vendor>({
   VendorSince: '2023-10-05',
   OnlineMap: false,
   HasSmartphone: false,
-  HasBankAccount: false
+  HasBankAccount: false,
+  OpenPayments: [
+    {
+      Amount: 0,
+      AuthorizedBy: '',
+      Id: 0,
+      IsPayoutFor: [],
+      IsSale: false,
+      Item: 0,
+      Order: 0,
+      OrderEntry: 0,
+      Payout: 0,
+      Price: 0,
+      Quantity: 0,
+      Receiver: 0,
+      ReceiverName: '',
+      Sender: 0,
+      SenderName: '',
+      Timestamp: ''
+    }
+  ]
```

Comment:
the openPayments can and should be empty while creating a new vendor. so the type here is wrong.